readthedocs/projects/models.py

```diff
@@ -1465,14 +1469,105 @@ def __str__(self):
         return self.email
 
 
-class WebHook(Notification):
+class WebHookEvent(models.Model):
+
+    BUILD_TRIGGERED = 'build:triggered'
+    BUILD_PASSED = 'build:passed'
+    BUILD_FAILED = 'build:failed'
+
+    EVENTS = (
+        (BUILD_TRIGGERED, _('Build triggered')),
+        (BUILD_PASSED, _('Build passed')),
```

Comment:
I think we are using succeed/successful mainly in the code. May be good to not introduce another way to call the same thing 😄

Reply:
This is what we use in the build pages

![Screenshot 2021-09-28 at 15-01-09 Builds Read the Docs](https://user-images.githubusercontent.com/4975310/135157409-b40627f8-c777-4f6d-8faf-ec0e7a08e3ed.png)

Reply:
Yeah, but internally we are using https://github.com/readthedocs/readthedocs.org/blob/c36aaf8fe7adb9153e0a2463e8b61f76826eb69c/readthedocs/builds/constants.py#L70-L72

So, I'd suggest keeping the same names internally.